src/OpenFeature.Contrib.Providers.Flagd/Resolver/InProcess/InProcessResolver.cs

```diff
@@ -97,14 +97,13 @@ private async void HandleEvents(CountdownEvent latch)
                     // Read the response stream asynchronously
                     while (await call.ResponseStream.MoveNext(token))
                     {
+                        var response = call.ResponseStream.Current;
+                        _evaluator.Sync(FlagConfigurationUpdateType.ALL, response.FlagConfiguration);
```

Comment:
I think this was a race - we we signaling the latch before we read, which caused some issues.